client/example_object_test.go

```diff
@@ -0,0 +1,243 @@
+	"github.com/nspcc-dev/neofs-sdk-go/user"
+)
+
+// ExampleClient_ObjectGetInit demonstrates how to initialize an object retrieval
+// operation using the NeoFS SDK for Go.
+func ExampleClient_ObjectGetInit() {
+	ctx := context.Background()
+
+	// Load wallet and decrypt account
+	w, err := wallet.NewWalletFromFile("wallet.json") // path to your wallet file
+	if err != nil {
+		panic(err)
+	}
+	addr := w.GetChangeAddress()
+	acc := w.GetAccount(addr)
+	err = acc.Decrypt("password", keys.NEP2ScryptParams())
+	if err != nil {
+		panic(err)
+	}
+	signer := user.NewAutoIDSigner(acc.PrivateKey().PrivateKey)
+
+	// Create and configure client
+	var prmInit client.PrmInit
+	c, err := client.New(prmInit)
+	if err != nil {
+		panic(fmt.Errorf("client init: %w", err))
+	}
+
+	// Connect to NeoFS node
+	var prmDial client.PrmDial
+	prmDial.SetServerURI("grpc://localhost:8080") // storage endpoint address
+	prmDial.SetTimeout(15 * time.Second)
+	prmDial.SetStreamTimeout(15 * time.Second)
+
+	if err = c.Dial(prmDial); err != nil {
+		panic(fmt.Errorf("dial: %w", err))
+	}
+
+	// Parse container ID from string
+	var containerID cid.ID
+	err = containerID.DecodeString("5HwTicwHLs3lc6RG8vAGSy4e41WwNd4RGMKKvF5pKYjC")
+	if err != nil {
+		panic(fmt.Errorf("parse container ID: %w", err))
+	}
+
+	// Parse object ID from string
+	var objectID oid.ID
+	err = objectID.DecodeString("3nQH1L7dQFKTjZFWpJeV1KSG7h1JcEPkZ6VhH5qZjQmW")
+	if err != nil {
+		panic(fmt.Errorf("parse object ID: %w", err))
+	}
+
+	// Configure get parameters
+	var prm client.PrmObjectGet
+	// Optional: mark as local if you want to read from local node only
+	// prm.MarkLocal()
+	// Optional: mark as raw to get the physical storage object
+	// prm.MarkRaw()
+
+	// Initiate object retrieval
+	objHeader, reader, err := c.ObjectGetInit(ctx, containerID, objectID, signer, prm)
+	if err != nil {
+		panic(fmt.Errorf("init object get: %w", err))
+	}
+	defer func() { _ = reader.Close() }()
+
+	// Access object header information
+	fmt.Printf("Object owner: %s\n", objHeader.Owner())
+	fmt.Printf("Payload size: %d bytes\n", objHeader.PayloadSize())
+	fmt.Println("Attributes:")
+	for _, v := range objHeader.Attributes() {
+		fmt.Printf("%s: %s\n", v.Key(), v.Value())
+	}
+	// Example output:
+	// Object owner: NfeCA7AuP5zodtJEPsSdkZ2gB3hKdNEhUg
+	// Payload size: 1024 bytes
+	// Attributes:
+	// FileName: example.txt
+	// ContentType: text/plain
+
+	// Read the payload
+	payload, err := io.ReadAll(reader)
+	if err != nil {
+		panic(fmt.Errorf("read payload: %w", err))
+	}
+
+	fmt.Printf("Successfully read %d bytes\n", len(payload))
+	// Example output: Successfully read 1024 bytes
+}
+
+// ExampleClient_ObjectPutInit demonstrates how to initialize an object upload
+// operation using the NeoFS SDK for Go.
+func ExampleClient_ObjectPutInit() {
+	ctx := context.Background()
+
+	// Load wallet and decrypt account
+	w, err := wallet.NewWalletFromFile("wallet.json") // path to your wallet file
```

Comment:
i'd move const wallet path/password, net/object addresses etc. to the top. And comment an invitation to set them

endpoint may be prefilled with some Testnet SN. I also thought that container ID may be prefilled w/ some public-write Testnet container